src/mscorlib/mscorlib.csproj

```diff
@@ -181,7 +181,8 @@
   </PropertyGroup>
 
   <Import Project="$(MSBuildThisFileDirectory)Tools\Versioning\GenerateVersionInfo.targets"/>
-  <Import Project="GenerateSplitStringResources.targets"/>
+  <Import Project="GenerateSplitStringResources.Windows.targets" Condition="'$(OsEnvironment)'=='Windows_NT'"/>
```

Comment:
Can we please have a single targets file and depending upon whether we are in open build or not, we should use resgen (for TFS build) or  python (for both Windows and Unix open builds) to generate the CS file?